Sources/UIView.swift

```diff
@@ -140,7 +140,10 @@ open class UIView: UIResponder, CALayerDelegate, UIAccessibilityIdentification {
 
     public init(frame: CGRect) {
         self.layer = type(of: self).layerClass.init()
-        self.layer.contentsScale = UIScreen.main.scale
+        if let uiScreenMain = UIScreen.main {
+            self.layer.contentsScale = uiScreenMain.scale
+        }
```

Comment:
I agree that this will probably prevent the crash from happening, but this might also break the layouting when `self.layer.contentsScale` has an invalid value when an UIView gets rendered. I think a more sustainable fix would be to ensure UIScreen.main exists before the first view gets rendered.

Did you manage to reproduce the crash locally @rikner ?

Reply:
I agree it would break layouting, so this is rather just a hotfix. I couldn't reproduce it locally, but I dont see anything else that could chrash during `UIView.init()`.

Reply:
Is there any stacktrace on bugsnag which makes us think that UIView.init() crashes the app? @rikner

Reply:
@michaelknoch [this one](https://app.bugsnag.com/flowkey-gmbh/mobile-app/errors/5e83408b045d4a0017329c48?filters[event.since][0][type]=eq&filters[event.since][0][value]=30d&filters[os.name][0][type]=eq&filters[os.name][0][value]=android&filters[error][0]=5e83408b045d4a0017329c48):

Reply:
Is it possible we're initing UIScreen, changing the orientation (which may deinit UIScreen temporarily) and then initing it again after? If so can we make sure the deinit-reinit thing happens synchronously? If so we can't crash at that point